rakam/src/main/java/org/rakam/analysis/ProjectHttpService.java

```diff
@@ -260,6 +261,19 @@ public SuccessMessage revokeApiKeys(@ApiParam("project") String project, @ApiPar
         return SuccessMessage.success();
     }
 
+    @JsonRequest
+    @ApiOperation(value = "Get possible attribute values",
+            authorizations = @Authorization(value = "read_key"))
+    @Path("/attributes")
+    public List<String> attributes(@ApiParam("project") String project,
+                                   @ApiParam("collection") String collection,
+                                   @ApiParam("field") SchemaField field,
```

Comment:
The parameter should also be `String` and renamed to `attribute`.

Reply:
Done